.travis.yml

```diff
@@ -8,6 +8,10 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.6-dev"
+  - "3.7-dev"
```

Comment:
Thanks for the super-quick feedback. Your comment's totally right. I can
 - update the PR to remove 3.7 / nightly
 - update the PR to add 3.7 / nightly to the allowed failures
 - close the PR.